### PR TITLE
Fix syntax for listing floating IPs

### PR DESCRIPTION
**Pull request by abel-navarro**

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -4289,7 +4289,7 @@ function oncontroller_testpreupgrade
 function oncontroller_testpostupgrade
 {
     # retrieve the ping results
-    local fips=$(openstack floating ip list -f value -c "Floating IP Address")
+    local fips=$(openstack ip floating list -f value -c IP)
```
Comment (s-t-e-v-e-n-k) [inline scripts/qa_crowbarsetup.sh]: This may require a guard, since it's 'ip floating' for Cloud 6, and 'floating ip' for Cloud 7.

Comment (abel-navarro) [inline scripts/qa_crowbarsetup.sh]: thanks for pointing out @s-t-e-v-e-n-k . I found this problem during the post upgrade step so I'm wondering if the controller did not got upgraded so it failed here. I'm trying to reproduce it to investigate further.

Comment (jsuchome) [inline scripts/qa_crowbarsetup.sh]: This ("Floating IP Address") code was indeed there for Cloud7. Your original did not work.

**Comment by abel-navarro** [comment]
So the error was in fact the the controllers were not upgraded at all, but the cloudupgrade step finished successfully. I'll focus on why that happened. This means this PR is no longer valid. Closing.

**Pull request closed without merging**